client/navigation/components/header/index.js

```diff
@@ -23,20 +24,30 @@ const Header = () => {
 		document.body.classList.toggle( 'is-folded' );
 	};
 
-	const foldOnMobile = ( screenWidth ) => {
-		if ( screenWidth <= 960 ) {
-			document.body.classList.add( 'is-folded' );
-		}
+	const foldOnMobile = ( screenWidth = document.body.clientWidth ) => {
+		const isSmallScreen = screenWidth <= 960;
+
+		document.body.classList[ isSmallScreen ? 'add' : 'remove' ](
+			'is-folded'
+		);
 	};
 
 	useEffect( () => {
-		foldOnMobile( document.body.clientWidth );
+		foldOnMobile();
+		const foldEvents = [
+			{
+				eventName: 'orientationchange',
+				handler: ( e ) => foldOnMobile( e.target.screen.availWidth ),
+			},
+			{
+				eventName: 'resize',
+				handler: debounce( () => foldOnMobile(), 200 ),
```

Comment:
Is there a reason to use the `debounce ` here?

Reply:
Just a small performance tweak. While dragging the resize handle on the app, the browser will rapid fire the resize event. Since we're really only interested in executing the callback after we're done resizing we can enforce that with `debounce()`